src/app/util/uploadToCloudinary.js

```diff
@@ -26,16 +26,22 @@ export const saveImage = (page, imageUrl) => {
     });
 };
 
-const uploadImage = (page, imageData) => {
-  axios
-    .post(CLOUDINARY_URL, imageData, {
+const uploadImage = (page, imageUrl) => {
+  const formData = new FormData();
+  formData.append('tags', page);
```

Comment:
I do not think we need this here again. Pull in develop and update.

Reply:
ok